src/Tools/Test Framework/Test Runner/src/ALTestRunnerResetEnvironment.Codeunit.al

```diff
@@ -48,36 +48,61 @@ codeunit 130453 "ALTestRunner Reset Environment"
 #pragma warning restore AA0207
 #else
     [EventSubscriber(ObjectType::Codeunit, Codeunit::"Test Runner - Mgt", 'OnAfterCodeunitRun', '', false, false)]
-    local procedure AfterTestMethod(var TestMethodLine: Record "Test Method Line")
```

Comment:
I'm wondering if perhaps this was supposed to subscribe to OnAfterTestMethodRun based on its name..?